searx/settings.yml

```diff
@@ -464,6 +464,11 @@ engines:
     shortcut : oap
     timeout: 5.0
 
+  - name : opensemanticsearch
```

Comment:
Engines which require action (installation, setting base_url) before being usable are usually commented out. See yacy as an example. Could you please comment it out?